app/src/test/java/org/mozilla/tv/firefox/fxa/FxaRepoTest.kt

```diff
@@ -0,0 +1,74 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.tv.firefox.fxa
+
+import android.content.Context
+import io.mockk.MockKAnnotations
+import io.mockk.impl.annotations.MockK
+import io.mockk.verify
+import io.reactivex.observers.TestObserver
+import io.reactivex.subjects.BehaviorSubject
+import mozilla.components.concept.sync.OAuthAccount
+import mozilla.components.concept.sync.Profile
+import mozilla.components.service.fxa.manager.FxaAccountManager
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.robolectric.RobolectricTestRunner
+
+@RunWith(RobolectricTestRunner::class)
+class FxaRepoTest {
+
+    @MockK private lateinit var context: Context
+    @MockK(relaxed = true) private lateinit var accountManager: FxaAccountManager
+    @MockK private lateinit var account: OAuthAccount
+    @MockK private lateinit var profile: Profile
```

Comment:
Nit: `profile`, `context`, and `account` are only used in one function each, could we move their setup to those funs?  The syntax for that would look like `val context = mockk<Context>()`.

It makes the test a little easier to read when there's less setup up top.